tools/barista/src/builder/overview.ts

```diff
@@ -41,6 +41,41 @@ const highlightedItems = [
   'Theming',
 ];
 
+function orderSectionItems(categoryNav: BaOverviewPage): BaOverviewPage {
+  for (const section of categoryNav.sections) {
+    const sectionItems = section.items;
+    if (section.title === 'Documentation') {
```

Comment:
Also other pages (not only documentation) can have the order property set. Sort all overview items the same way: check if both items have an order-value and compare this one, if not, sort by title, if only one has an order-value, this one should come first.